controllers/goroutines/operator_status.go

```diff
@@ -40,23 +40,16 @@ func UpdateCsCrStatus(bs *bootstrap.Bootstrap) {
 		}
 
 		var operatorSlice []apiv3.BedrockOperator
-		operatorsName := []string{
-			"ibm-auditlogging-operator",
-			"ibm-cert-manager-operator",
-			"ibm-commonui-operator",
+
+		operatorsName := []string{}
+		opreg := bs.GetOperandRegistry(ctx, "common-service", bs.CSData.MasterNs)
+		for i := range opreg.Spec.Operators {
+			operatorsName = append(operatorsName, opreg.Spec.Operators[i].Name)
+		}
+		operatorsName = append(operatorsName, []string{
 			"ibm-crossplane-operator-app",
-			"ibm-events-operator",
-			"ibm-healthcheck-operator",
-			"ibm-iam-operator",
-			"ibm-ingress-nginx-operator",
-			"ibm-licensing-operator",
-			"ibm-management-ingress-operator",
-			"ibm-mongodb-operator",
-			"ibm-monitoring-grafana-operator",
 			"ibm-namespace-scope-operator",
-			"ibm-platform-api-operator",
-			"ibm-zen-operator",
-			"operand-deployment-lifecycle-manager-app"}
+			"operand-deployment-lifecycle-manager-app"}...)
```

Comment:
Would we include the `ibm-cloud-operator` as well?

Reply:
actually, I am wondering will we add crossplane operator and cloud operator to operandregistry in the futurn? maybe @horis233

Reply:
No, they won't be added into operandregistry. We can add both of them to this list.

Reply:
thank you all! added to the list